source/startup.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+apt-get update -y
+apt-get install -y cron
+
+echo "* * * * * . /etc/profile ; /usr/local/bin/php /home/site/wwwroot/cron/cron.php" | crontab -
+
+service cron start
```

Comment:
I'm not really sure of the container / VM lifecycle on azure - but does this mean that it'll do a full apt package list update and attempt to re-install cron every time the container restarts?

Normally I'd expect that to happen once at build time, and the cron service to start at runtime, but not at build?

Is the azure VM getting rebuilt every time?

If so, maybe worth including the other requirements, which are only listed in the `Dockerfile` at the moment.

Reply:
Also, does this mean that the `cron.php` is being run by the system `root` user, rather than by the `site` or PHP user?